py4web/utils/form.py

```diff
@@ -280,6 +295,29 @@ def FormStyleBulma(table, vars, errors, readonly, deletable):
     return FormStyleDefault(table, vars, errors, readonly, deletable, classes)
 
 
+def FormStyleBootStrap(table, vars, errors, readonly, deletable):
```

Comment:
Maybe FormStyleBootStrap4? We run into this issue for web2py

Reply:
makes sense, has been changed.